morph/react/property-event-handler.js

```diff
@@ -0,0 +1,23 @@
+import { getImportNameForSource, getVariableName } from '../utils.js'
+
+export function enter(node, parent, state) {
+  if (
+    !/^on[A-Z]/.test(node.name) ||
+    node.name === 'onWhen' ||
+    node.tags.slot ||
+    typeof node.value !== 'string' ||
+    !node.source
+  ) {
+    return
+  }
+
+  let importName = getImportNameForSource(node.source, state)
+  // assuming it is a hook
+  if (node.value.startsWith('use')) {
```

Comment:
what about checking for `/^use[A-Z]/` instead so we don't leave out `use` as a prefix for other things?